web/src/api.js

```diff
@@ -83,9 +83,22 @@ function getLoggedUser () {
     })
 }
 
+function getConfigProjectDefinitions(pathname) {
+  /* Return json list that include project definitions from config file */
+  const indexname = pathname.replace("/", "");
+  const url = baseurl + "/projects?index=" + indexname;
+  return axios
+    .get(url)
+    .then((response) => response.data)
```

Comment:
The `api.js` module does not process the response or catch error in the other function. It seems better to push that work downstream, is there a reason `getConfigProjectDefinitions` handle the axios promise?